library/core/src/mem/mod.rs

```diff
@@ -214,7 +214,8 @@ pub fn forget_unsized<T: ?Sized>(t: T) {
 /// The alignment of the struct is usually the largest alignment of all its
 /// fields; this can be changed with the use of `repr(align(N))`.
 ///
-/// Unlike `C`, zero sized structs are not rounded up to one byte in size.
+/// Unlike `C`'s `sizeof` operator, zero sized structs are not rounded up to
```

Comment:
Talking about `sizeof` seems confusing. C structs have a fixed layout regardless of whether or not you use sizeof.